xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4520,6 +4527,9 @@ bool CVideoPlayer::OnAction(const CAction &action)
             return true;
           }
         }
+        if (!channel) {
```

Comment:
Same as for the other case:

Please move the curly brace to next line (Kodi coding guidelines!).

No need for the <code>channel</code> variable anymore. You could write <code>if (!g_PVRManager.GetCurrentChannel())</code> instead.